mindera-people/Sources/HomeView/HomeView.swift

```diff
@@ -81,16 +81,16 @@ public struct HomeView: View {
     
     private func text(_ selectionValue: Home.Tab,
                       _ viewStore: ViewStore<Home.State, Home.Action>) -> some View {
-        var text = ""
+        var text: String.LocalizationValue = ""
         switch selectionValue {
         case .dashboard:
-            text = "Dashboard"
+            text = "dashboard"
         case .calendar:
-            text = "Calendar"
+            text = "calendar"
         case .myProfile:
-            text = "My Profile"
+            text = "my-profile"
         }
-        return Text(text)
+        return Text(String(localized: text))
```

Comment:
in SwiftUI, the Text component received by default a `LocalizedStringKey`. If there is not a key with this value, it will retrieve the key itself. So when we fill this initialiser is not a "pure" string.

With the implementation above we can clean this, and by so, drop the extra `String(localised:)`

```suggestion
        return Text(text)
```